src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/GregtechMetaTileEntity_QuantumForceTransformer.java

```diff
@@ -810,13 +810,15 @@ private boolean processRecipe(
                                     mat.getFluid(tRecipe.getOutput(i).stackSize * 1000);
                                 } else {
                                     Item aItem = tRecipe.getOutput(i).getItem();
+                                    int aDamage = tRecipe.getOutput(i).getItemDamage();
                                     int aStackSize = tRecipe.getOutput(i).stackSize;
-                                    tItemOutputs.add(new ItemStack(aItem, aStackSize * mCurrentParallel));
+                                    tItemOutputs.add(new ItemStack(aItem, aStackSize * mCurrentParallel, aDamage));
```

Comment:
What about `GT_Utility.copyAmount`?

Reply:
GT_Utility.copyAmount, limits it to 64 in a stack still, but i realized another way i can do it. will push it

Reply:
use `GT_Utility.copyAmountUnsafe`

Reply:
fixed i think